aptos-move/framework/aptos-framework/sources/genesis.spec.move

```diff
@@ -61,6 +90,16 @@ spec aptos_framework::genesis {
         include InitalizeRequires;
     }
 
+    spec set_genesis_end {
```

Comment:
Could you combine this with the other spec block of set_genesis_end in this file?

Reply:
sure